pycsob/client.py

```diff
@@ -219,7 +218,8 @@ def customer_info(self, customer_id):
         r = self._client.get(url)
         return utils.validate_response(r, self.pubkey)
 
-    def oneclick_init(self, orig_pay_id, order_no, total_amount, currency='CZK', description=None):
+    def oneclick_init(self, orig_pay_id, order_no, total_amount, currency='CZK', description=None,
+                      return_url='http://localhost', return_method='GET', client_initiated=False):
```

Comment:
Parameters `return_url` and `return_method` are mandatory, but in fact are not needed, when `client_initiated=False`. See https://github.com/csob/platebnibrana/issues/26.